torchvision/models/detection/_utils.py

```diff
@@ -300,6 +300,32 @@ def decode_single(self, rel_codes: Tensor, boxes: Tensor) -> Tensor:
         pred_boxes = torch.stack((pred_boxes1, pred_boxes2, pred_boxes3, pred_boxes4), dim=1)
         return pred_boxes
 
+    def decode_all(self, list_rel_codes: Tensor, list_boxes: List[Tensor]) -> Tensor:
+        if len(list_rel_codes) != len(list_boxes):
+            min_len = min(len(list_rel_codes), len(list_boxes))
+            list_boxes, list_rel_codes = list_boxes[:min_len], list_rel_codes[:min_len]
+        list_boxes = torch.stack(list_boxes)
+
+        list_boxes = list_boxes.to(list_rel_codes.dtype)
+
+        list_ctr_x = 0.5 * (list_boxes[..., 0] + list_boxes[..., 2])
+        list_ctr_y = 0.5 * (list_boxes[..., 1] + list_boxes[..., 3])
+
+        if self.normalize_by_size:
+            list_boxes_w = list_boxes[..., 2] - list_boxes[..., 0]
+            list_boxes_h = list_boxes[..., 3] - list_boxes[..., 1]
+
+            list_box_size = torch.stack((list_boxes_w, list_boxes_h, list_boxes_w, list_boxes_h), dim=2)
```

Comment:
I believe the following is left-over from when you used `[:, :]` instead of `...`. To avoid assuming a specific number of dimensions, I think we can do:
```suggestion
            list_box_size = torch.stack((list_boxes_w, list_boxes_h, list_boxes_w, list_boxes_h), dim=-1)
```

Reply:
sounds good! Thanks @datumbox !